app/src/main/java/com/vultisig/wallet/presenter/keygen/KeygenPeerDiscovery.kt

```diff
@@ -92,7 +94,7 @@ fun KeygenPeerDiscovery(
         )
         Spacer(modifier = Modifier.height(MaterialTheme.dimens.small2))
         if (viewModel.keygenPayloadState.value.isNotEmpty()) {
-            QRCodeKeyGenImage(viewModel.keygenPayloadState.value)
+            QRCodeKeyGenImage(viewModel.keygenPayloadState.value, (LocalConfiguration.current.screenWidthDp/2).toInt().dp,(LocalConfiguration.current.screenWidthDp/2).toInt().dp)
```

Comment:
is it trying to make it half of the screen size?  we want it to be as big as possible actually